src/main/scala/com/lightbend/conductr/sbt/BundlePlugin.scala

```diff
@@ -307,9 +307,12 @@ object BundlePlugin extends AutoPlugin {
     }
 
   private def formatAcl(acl: RequestAcl): String =
-    s"""          {
-        |${acl.protocolFamilyRequestMappings.map(formatRequestMappings).mkString(",\n")}
-        |          }""".stripMargin
+    if (acl.protocolFamilyRequestMappings.isEmpty)
+      ""
+    else
+      s"""          {
+          |${acl.protocolFamilyRequestMappings.map(formatRequestMappings).mkString(",\n")}
+          |          }""".stripMargin
```

Comment:
@huntc - brace removed, can't use `fold` since `acl.protocolFamilyRequestMappings` is a `Set[ProtocolFamilyRequestMapping]`